furnace.c

```diff
@@ -230,6 +242,17 @@ command_handler(furnace_context_t* ctx, uint8_t* buffer, void (*feedback)(const
     }
   }
 #endif
+#if CONFIG_WATER
+  else if(sscanf(buffer, "water %u", &arg) == 1) {
+    if (set_pwm_safe(WATER_PIN, ctx, arg) == 0){
+      ctx->pilot.is_enabled = 0;
```

Comment:
We disable the pilot when changing water pwm duty?

Reply:
Yeah, we shouldn't.